README.md

```diff
@@ -1,91 +1,95 @@
-# EVM Placeholder Proof System Verifier 
+# EVM Placeholder proof system verifier 
 
 [![Discord](https://img.shields.io/discord/969303013749579846.svg?logo=discord&style=flat-square)](https://discord.gg/KmTAEjbmM3)
 [![Telegram](https://img.shields.io/badge/Telegram-2CA5E0?style=flat-square&logo=telegram&logoColor=dark)](https://t.me/nilfoundation)
 [![Twitter](https://img.shields.io/twitter/follow/nil_foundation)](https://twitter.com/nil_foundation)
 
-This repository contains the smart contracts for validating zero knowledge proofs 
-generated in placeholder proof system in EVM. 
+This repository contains the smart contracts for validating zero-knowledge proofs generated
+via [Placeholder proof system](https://github.com/NilFoundation/evm-placeholder-verification) in-EVM.
```

Comment:
1. You don't need the "this repository" part, because the first sentence obviously describes the contents of the repository.
2. As far as I remember, we prefer "EVM apps" in favor of "smart contracts".

```suggestion
EVM application for validating zero-knowledge proofs generated with the
[Placeholder proof system](https://github.com/NilFoundation/evm-placeholder-verification).
```